rpc/wrtc_signaling_answerer.go

```diff
@@ -400,6 +400,9 @@ func (aa *answerAttempt) connect(ctx context.Context) (err error) {
 		}
 	}()
 
+	// `NewChannel` instantiates our webrtc wrapper around DataChannels. This includes callback
+	// handlers for transitioning to the open/closed/error states. As well as backpressure when the
+	// amount of data to send gets high.
```

Comment:
> backpressure when the amount of data to send gets high

Is this referring to the buffer threshold? What is "backpressure"?

Reply:
Yes, the buffer threshold is backpressure.

Backpressure is when a function, after determining it's using too many resources, either:
* fails back to the caller saying "i'm too busy" or
* blocks until there are more available resources before processing the function

call tickets is another example of backpressure.